src/Alert/Alert.component.tsx

```diff
@@ -0,0 +1,178 @@
+): FlattenInterpolation<ThemeProps<AlertProps>> => {
+    return css`
+        .anchor-alert-icon {
+            ${({ theme }) =>
+                css({
+                    color: darken(
+                        0.2,
+                        get(theme, `colors.${variant.toLowerCase()}`)
+                    ),
+                })}
+        }
+    `;
+};
+
+const StyledAlert = styled('div')<AlertProps>`
+    box-sizing: border-box;
+    position: relative;
+    border-radius: base;
+    font-family: ${th('typography.fontFamily')};
+    padding: 0.5rem 1.5rem 0.5rem 0.5rem;
+    ${({ variant }) =>
+        css({
+            backgroundColor: `background.${variant.toLowerCase()}`,
+            border: 'solid thin',
+            borderColor: `borders.${variant.toLowerCase()}`,
+        })};
+    ${({ variant }) => renderAlertTheme(variant)};
+
+    .anchor-alert-close {
+        position: absolute;
+        top: 0;
+        right: 0;
+
+        &:active,
+        &:hover,
+        &:focus {
+            background-color: transparent;
+        }
+    }
+
+    h4,
+    p {
+        line-height: 1.2em;
+        margin: 0;
+        padding: 0;
+    }
+
+    h4 {
+        padding-bottom: 0.75rem;
+        &:last-child {
+            padding-bottom: 0;
+        }
+    }
+`;
+
+const renderMessageAndDescription = ({
+    message,
+    description,
+}: {
+    message?: string;
+    description?: string;
+}) => (
+    <>
+        {message && <Typography tag="h4">{message}</Typography>}
+        {description && (
+            <Typography tag="p" color="charcoal" hue="light">
+                {description}
+            </Typography>
+        )}
+    </>
+);
+
+const StyledIconContainer = styled('div')`
+    box-sizing: border-box;
+    display: flex;
+
+    .anchor-alert-content {
+        flex: 1 1 auto;
+    }
+    .anchor-icon {
+        flex: 0 0 0;
+        padding: 0 0.5rem 0 0;
+    }
+`;
+
+export const Alert = ({
+    className,
+    children,
+    onClose = () => null,
+    message,
+    description,
+    closable,
+    icon,
+    ...props
+}: AlertProps): React.ReactElement<AlertProps> | null => {
+    const [isHidden, setIsHidden] = React.useState<boolean>(false);
+
+    const handleClose = () => {
+        setIsHidden(true);
+        onClose();
+    };
+
+    return !isHidden ? (
+        <StyledAlert
+            className={classNames('anchor-alert', className)}
+            {...props}
+        >
+            {closable && (
+                <Button
+                    className="anchor-alert-close"
+                    size="xs"
+                    circular={true}
+                    variant="minimal"
+                    prefix={<Close color="rgba(0, 0, 0, 0.3)" scale="sm" />}
+                    onClick={handleClose}
+                />
+            )}
+            {icon ? (
+                <StyledIconContainer>
+                    {React.cloneElement(icon, {
+                        className: 'anchor-alert-icon',
```

Comment:
This is pretty edge case, but this would override the className if a user passed one in. It's one reason I made the `cloneWithProps` util. e.g.
```jsx
icon={<InfoOutline scale="md" className="qa-alert-icon" />}
```
`qa-alert-icon` class would be lost

Reply:
Ah, that does make more sense. Pushing a fix now. Not sure why our CI is crashing. I was having a git issue earlier and I'm wondering if that corrupted something..